models/omen/gnosis/omen_gnosis_liquidity.sql

```diff
@@ -0,0 +1,220 @@
+{{ config(
+    schema = 'omen_gnosis',
+    alias = 'liquidity',
+    
+    partition_by = ['block_day'],
+    materialized = 'incremental',
+    file_format = 'delta',
+    incremental_strategy = 'merge',
+    unique_key = ['block_time', 'tx_hash', 'evt_index'],
+    post_hook='{{ expose_spells(\'["gnosis"]\',
+                                "project",
+                                "omen",
+                                \'["hdser"]\') }}'
+    )
+}}
+
+{% set project_start_date = '2020-12-01' %}
+
+
+WITH 
+
+FPMMFundingAdded AS (
+    SELECT
+        t.block_time,
+        t.tx_from,
+        t.tx_to,
+        t.tx_hash,
+        t.evt_index,
+        t.evt_contract_address,
+        t.funder,
+        t.sharesMinted,
+        ARRAY_AGG(s.SEQUENCE_NUMBER - 1 ORDER BY s.SEQUENCE_NUMBER) AS outcomeIndex,
+        ARRAY_AGG(
+            VARBINARY_TO_UINT256(
+                VARBINARY_LTRIM(
+                    VARBINARY_SUBSTRING(t.data, 65 + 32 * s.SEQUENCE_NUMBER, 32)
+                )
+            ) ORDER BY s.SEQUENCE_NUMBER) AS outcomeTokens_amount
+    FROM (
+        SELECT
+            block_time,
+            tx_hash,
+            tx_from,
+            tx_to,
+            index AS evt_index,
+            contract_address AS evt_contract_address,
+            VARBINARY_SUBSTRING(topic1, 13, 20) AS funder,
+            VARBINARY_TO_UINT256(VARBINARY_LTRIM(VARBINARY_SUBSTRING(data, 33, 32))) AS sharesMinted,
+            VARBINARY_TO_UINT256(VARBINARY_LTRIM(VARBINARY_SUBSTRING(data, 65, 32))) AS amountsAdded_size,
+            data
+        FROM 
+            {{source('gnosis','logs') }}
+        WHERE
+            --FPMMDeterministicFactory_evt_FPMMFundingAdded
+            topic0 = 0xec2dc3e5a3bb9aa0a1deb905d2bd23640d07f107e6ceb484024501aad964a951 
+            {% if is_incremental() %}
+            AND block_time >= date_trunc('day', now() - interval '7' day)
```

Comment:
We can use `incremental_predicate` here and for other incremental filters:
```suggestion
            AND {{ incremental_predicate('block_time') }}
```

Also if you think this PR is ready-for-review, please mark it ready from draft status.